ui/admin/app/controllers/scopes/scope/workers/index.js

```diff
@@ -16,8 +17,42 @@ export default class ScopesScopeWorkersIndexController extends Controller {
   @service router;
   @service can;
 
+  // =attributes
+
+  @tracked selectedWorker;
+
+  /**
+   * Get the first 10 tags of the selected worker.
+   * @type {object[]}
+   */
+  get selectedWorkerTags() {
+    // only show the first 10 tags if there are more than 10
+    return this.selectedWorker.allTags.slice(0, 10);
+  }
+
+  /**
+   * Build the display name for a worker tag and will
+   * truncate the key if it's too long.
+   * @param {object} tag
+   * @returns {string}
+   */
+  tagDisplayName(tag) {
```

Comment:
I don't think we should truncate the actual text. We should accomplish this with `text-overflow: ellipsis` since if you try and copy and paste the key now, it's the truncated text but if you do it with CSS, you still get the full text.

Reply:
Right, `text-overflow` is what we have done in the past. Thanks.

Yep that is a good point. With that in mind, should we wrap the whole key/value pair with `text-overflow` or the key and value separately?

Reply:
It depends what we're going for, doing it separately means you can control how big the key and value are and independently overflow. I'm not sure if key or value might be more important to a user so I'm not sure what's a better UX but I think either could make sense

Reply:
Ok. I will play around with wrapping each separately. I think there is value in always trying to show the key and value.

Reply:
Updated. Let know what you think. Here is an example of what it looks like for the different cases: long key, long key and value, and long value.
<img width="995" alt="Screenshot 2024-06-21 at 3 24 14 PM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/c09c003a-80fe-4e24-899b-10e9d41e80d2">

Reply:
<img width="554" alt="Screenshot 2024-06-24 at 3 42 02 PM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/381cb328-1835-4ed7-8726-1f8f511fa52d">
Sharing again to show this is still displaying how we want after style change.